spec/unit/lib/cadence_spec.rb

```diff
@@ -206,6 +205,35 @@ class TestStartWorkflow < Cadence::Workflow
       end
     end
 
+    describe '.get_workflow_history' do
+      let(:cadence_response) do
+        CadenceThrift::StartWorkflowExecutionResponse.new(runId: 'xxx')
+      end
+      let(:response_mock) { double }
+      let(:history_mock) { double}
+      let(:event_mock) { double('EventMock', :eventId=> 1, :timestamp => Time.now.to_f, :eventType => 'ActivityTaskStarted', :eventAttributes => '') }
+  
+      before do
+        allow(client).to receive(:start_workflow_execution).and_return(cadence_response)
+        allow(history_mock).to receive(:events).and_return([event_mock])
+        allow(response_mock).to receive(:history).and_return(history_mock)
+      end
+
+      it 'wraps client get_workflow_execution_history' do
+          run_id = described_class.start_workflow(TestStartWorkflow, 42, options: { workflow_id: '123' })
+          expect(client).to receive(:get_workflow_execution_history).with(
```

Comment:
nit: I'm trying to separate stubbing from assertion whenever possible, can you please do the same? In practise it means using `allow().to receive().and_return` to stub response and then `expect().to have_received().with()` to check that it was called. This makes it easier to reuse these across multiple examples and is considered a good practise